python/its-vehicle/its_vehicle/tracking.py

```diff
@@ -0,0 +1,35 @@
+# Software Name: its-vehicle
+# SPDX-FileCopyrightText: Copyright (c) 2023 Orange
+# SPDX-License-Identifier: MIT
+# Author: Yann E. MORIN <yann.morin@orange.com>
+
+import logging
```

Comment:
Some imports: logging, time, paho.mqtt.client GNSSReport and MqqtClient are unused for the moment.